editor/src/components/canvas/commands/delete-properties-command.ts

```diff
@@ -24,19 +26,40 @@ export function deleteProperties(
   }
 }
 
-export const runDeleteProperties: CommandFunction<DeleteProperties> = (
+export const runDeleteProperties = (
   editorState: EditorState,
   command: DeleteProperties,
+  interactionLifecycle: InteractionLifecycle,
 ) => {
+  const propsToDelete = command.properties.reduce(
+    (acc: { styleProps: string[]; rest: PropertyPath[] }, p) => {
+      const prop = maybeCssPropertyFromInlineStyle(p)
+      if (prop == null) {
+        acc.rest.push(p)
+      } else {
+        acc.styleProps.push(prop)
+      }
+      return acc
+    },
+    { styleProps: [], rest: [] },
+  )
+
   // Apply the update to the properties.
-  const { editorStatePatch: propertyUpdatePatch } = deleteValuesAtPath(
+  const { editorStateWithChanges } = deleteValuesAtPath(
     editorState,
     command.element,
     command.properties,
```

Comment:
shouldn't this only run for `propsToDelete.rest`?

Reply:
great catch, fixed in https://github.com/concrete-utopia/utopia/pull/6599/commits/09086039ca30b2d337f1b424ce71355a0bbf980c